### PR TITLE
Explain Dlang-Bot's cross for references

### PR DESCRIPTION
**Pull request by wilzbach**
From https://github.com/dlang/phobos/pull/5543

### CODE REVIEW

**Comment by CyberShadow** [comment]
Good effort, but I doubt many people are going to sit down and read the documentation. I think UI changes would be more effective.

The table is a little cryptic, maybe a bullet list in plain English would be better, e.g.:

* Merging this pull request will close [issue 12345 - std.algorithm.searching.skipOver should have a single argument with pred version](...)
* Merging this pull request will **not** close [issue 12346 - std.algorithm.searching.skipOver yada bla bla bla](...)

**Comment by wilzbach** [comment]
> Good effort, but I doubt many people are going to sit down and read the documentation. I think UI changes would be more effective.
> The table is a little cryptic, maybe a bullet list in plain English would be better, e.g.:

The table has been there for a long and been done by @MartinNowak, so I don't want to change this without his approval (and I actually like the table), but there's a quick fix:

https://github.com/dlang-bots/dlang-bot/pull/131

**Comment by ZombineDev** [comment]
I also like the table, FWIW.
